package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-js-sdk",
-  "version": "2.19.0",
+  "version": "2.19.1-alpha.02",
```

Comment:
Probably not a hotfix, I would consider it an added feature so a minor version bump is required.